test/nbrowser/HomeIntro.ts

```diff
@@ -41,6 +41,49 @@ describe('HomeIntro', function() {
     it('should allow collapsing examples and remember the state', testExamplesCollapsing);
     it('should show examples workspace with the intro', testExamplesSection);
     it('should render selected Examples workspace specially', testSelectedExamplesPage);
+
+    describe('With GRIST_ANON_PLAYGROUND set to false', function () {
+      gu.withEnvironmentSnapshot({GRIST_ANON_PLAYGROUND: false});
```

Comment:
There's a bit of an awkward issue here.

If you look in `test/deployment` you'll see that a selection of tests are imported there, including `HomeIntro`. These tests are used as a way to *remotely* check if a deployment of Grist is sane. In `grist-core` you can see these tests used in `test/test_under_docker.sh`, where a Grist server is run in a docker container, and then tests are run with the only access to the server being via the network.

Since your test relies on reconfiguring the server, it cannot be used as a deployment test. So you'll need to move it to a different file.

(I see earlier in this file there is a `gu.withEnvironmentSnapshot({'GRIST_TEMPLATE_ORG': 'templates'});` which happens to work also in deployment tests since that is exactly the value set in the docker container also)

Reply:
Thanks! That should be done.

Also I realize that I don't test how the home intro is rendered when a user is logged in. I don't know how worth that is (if it is we should consider factorize with `HomeIntro.ts`).